helm/thingsboard/templates/node.yaml

```diff
@@ -170,6 +171,21 @@ spec:
       affinity:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.node.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- else }}
+      affinity:
+        {{- if .Values.node.podAffinityPreset }}
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.node.podAffinityPreset "customLabels" $podLabels "context" $) | nindent 10 }}
+        {{- end }}
+        {{- if .Values.node.podAntiAffinityPreset }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.node.podAntiAffinityPreset "customLabels" $podLabels "context" $) | nindent 10 }}
+        {{- end }}
+        {{- if .Values.node.nodeAffinityPreset }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.node.nodeAffinityPreset.type "key" .Values.nodeAffinityPreset.key "values" .Values.nodeAffinityPreset.values) | nindent 10 }}
```

Comment:
What is 'nodeAffinity' ?

Reply:
If I remember it correctly is the preference of a pod to run in a given node. For example a node with labels that indicates that it have a faster hard disk or AI accelerator could be preferred for some kind of pods.

Reply:
Ok, so we don't use this for now, right?